test/es6/HTMLComments.baseline

```diff
@@ -0,0 +1,13 @@
+Code before CRLF--> is reachable
+Code before CR--> is reachable
+Code before LF--> is reachable
+Code before LS--> is reachable
+Code before PS--> is reachable
+Code before CRLS--> is reachable
+Code before CRPS--> is reachable
+Code before <!-- is reachable
+Code before <!-- --> is reachable
+Code before <!-- LineTerminator --> is reachable
+Code before /* */ is reachable
+Code before /* */ is reachable

```

Comment:
No, the test is for the single line comment variant /* */ --> which should be the test description.